stage #1/module #1. Intro, Infrastructure & Instruments, Fundamentals/lesson #3. Classes, Intro to OOP/Weapon/src/test/java/com/epam/training/weapons/model/BladeTest.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.training.weapons.model;
+
+import org.junit.Test;
+
+import static com.epam.training.weapons.model.Blade.HandCount.TWO_HAND;
+import static org.junit.Assert.*;
+
+import static com.epam.training.weapons.model.Blade.HandCount.ONE_HAND;
+
+public class BladeTest {
```

Comment:
and we usually don't test model classes since setters and getters are too simple to test

Reply:
I will leave tests to gain 100% of coverage